cli/magic-config.ts

```diff
@@ -13,6 +13,8 @@ import {
 } from "../lib/shared/types";
 import { LIB_VERSION } from "./version.js";
 import * as fs from "fs";
+import { AWSCronValidator } from "./aws-cron-validator"
+import { AWSCronValidatorSimple } from "./aws-cron-validator-simple"
```

Comment:
I assume this line can be killed off since as I don't see a reference to that class anywhere?